latexify/core.py

```diff
@@ -36,11 +36,15 @@ def generic_visit(self, node):
   def visit_Module(self, node):
     return self.visit(node.body[0])
 
-  def visit_FunctionDef(self, node):
+  def visit_FunctionDef(self, node, equal:str=r'\triangleq'):
     name_str = r'\operatorname{' + str(node.name) + '}'
     arg_strs = [self._parse_math_symbols(str(arg.arg)) for arg in node.args.args]
     body_str = self.visit(node.body[0])
-    return name_str + '(' + ', '.join(arg_strs) + r') \triangleq ' + body_str
+    # CHANGE:(20200801@1MLightyears)
```

Comment:
Remove all `CHANGE:` comments to keep the code simple. This kind of comments are unnecessary since commit messages should provide enough information.

Reply:
for "CHANGE"
ah... sorry to bring in my personal habit. I used to work without git.
Will be removed in the next commit.